### DISCUSSION TITLE
Corkey Corrupt extensions Without Powerwash

### DISCUSSION THREAD

**Discussion by Inglan2** [3kh0/ext-remover#1042] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
If you don't want to powerwash or your sysadmin has to enroll it themselves and you cant do it yourself. Change your password and when you log in press forgot old password, then clear local data. this will only work sometimes

**Comment by Blobby-Boi** [comment]
what
Didn't you just say without powerwashing????

**Reply by Inglan2** [reply]
whoops typo. edited it
